lib/bolt/cli.rb

```diff
@@ -584,22 +584,14 @@ def initialize_project
     end
 
     def migrate_project
-      if inventory.version == 2
-        ok = true
-      else
-        inventory_file = config.inventoryfile || config.boltdir.inventory_file
-
-        begin
-          Bolt::Util.file_stat(inventory_file)
-        rescue Errno::ENOENT
-          raise Bolt::FileError.new("The inventory file '#{inventory_file}' does not exist", inventory_file)
-        end
+      inventory_file = config.inventoryfile || config.default_inventoryfile
+      data = Bolt::Util.read_yaml_hash(inventory_file, 'inventory')
 
-        inv = YAML.safe_load(File.open(inventory_file))
-        migrate_group(inv)
-
-        ok = File.write(inventory_file, { 'version' => 2 }.merge(inv).to_yaml)
-      end
+      ok = if migrate_group(data)
```

Comment:
We should report that there was nothing to do if `migrate_group` returns false rather than saying we migrated it when we didn't.